src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/config/InboundEventIntegrationConfig.java

```diff
@@ -0,0 +1,187 @@
+package uk.gov.ons.ctp.integration.contactcentresvc.config;
+
+import org.springframework.amqp.core.MessageProperties;
+import org.springframework.amqp.rabbit.config.StatelessRetryOperationsInterceptorFactoryBean;
+import org.springframework.amqp.rabbit.connection.ConnectionFactory;
+import org.springframework.amqp.rabbit.listener.SimpleMessageListenerContainer;
+import org.springframework.amqp.rabbit.retry.RejectAndDontRequeueRecoverer;
+import org.springframework.amqp.support.converter.DefaultClassMapper;
+import org.springframework.amqp.support.converter.Jackson2JsonMessageConverter;
+import org.springframework.amqp.support.converter.MessageConverter;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.integration.amqp.inbound.AmqpInboundChannelAdapter;
+import org.springframework.integration.channel.DirectChannel;
+import org.springframework.messaging.MessageChannel;
+import org.springframework.retry.RetryPolicy;
+import org.springframework.retry.backoff.ExponentialBackOffPolicy;
+import org.springframework.retry.support.RetryTemplate;
+import org.springframework.util.backoff.BackOff;
+import org.springframework.util.backoff.ExponentialBackOff;
+import uk.gov.ons.ctp.common.event.model.CaseEvent;
+import uk.gov.ons.ctp.common.jackson.CustomObjectMapper;
+import uk.gov.ons.ctp.common.retry.CTPRetryPolicy;
+import uk.gov.ons.ctp.integration.contactcentresvc.config.MessagingConfig.ContainerConfig;
+
+/** Integration configuration for inbound events. */
+@Configuration
+public class InboundEventIntegrationConfig {
+
+  private AppConfig appConfig;
+
+  /**
+   * Constructor for InboundEventIntegrationConfig
+   *
+   * @param appConfig centralised configuration
+   */
+  public InboundEventIntegrationConfig(final AppConfig appConfig) {
+    this.appConfig = appConfig;
+  }
+
+  /**
+   * Configure a backoff for what happens when rabbitMQ goes down. This allows us to do less
+   * thrashing than the default when there is a rabbit problem thus reducing load on the system.
```

Comment:
Minor, but this comment is a bit misleading. It sounds like it's more about reducing the retry frequency to keep the logs quiet. Can't image that a reconnection attempt generates any measurable load.

Reply:
I'm reluctant to change this since it mirrors the RHSvc code.  
I suggest if this looks code carries on we find a way to common up the RHSvc and CCsvc code, and address the javadoc then.